pkg/server/helpers.go

```diff
@@ -391,3 +392,18 @@ func resolveSymbolicLink(path string) (string, error) {
 	}
 	return filepath.EvalSymlinks(path)
 }
+
+func initSelinuxOpts(selinuxOpt *runtime.SELinuxOption) (string, string, error) {
+	var labelOpts []string
```

Comment:
nit: Move this down.

Reply:
And this seems cleaner https://github.com/opencontainers/selinux/blob/63bd19516561f1973f86c9f777b263ee1e0db140/go-selinux/selinux.go#L572

label.InitLabels(label.DupSecOpt("user:role:type:level")).